src/component/Carousel/Carousel.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Carousel } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import "./Carousel.css";
+
+export function Carouselbar() {
+  return (
+    <div className="home" id="home">
+    <div className="slide">
+    <Carousel fade>
+      <Carousel.Item className="image">
+        <img
+          className="travel"
+          src={require("../../assets/images/trekking.png")}
```

Comment:
You should write all the import/require statements at the start of your code.

Bring this line to the top, you can store them in variables and use them here.

eg.
const trekkingImg = require(../../assets/images/trekking.png)
import trekkingImg from "../../assets/images/trekking.png"